tests/ui/js/base-page-storyEditTests.js

```diff
@@ -863,6 +879,303 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "audio",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: "Not really a URL"
+            },                  blockContentValues: "default",  expectedEmpty: false
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "An actual heading",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "Some alternative text",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "A real description",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "A real transcript",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: "Not really a URL"
+            },                  blockContentValues: "default",  expectedEmpty: false
+        }
+    ];
+
+    sjrk.storyTelling.base.page.storyEditTester.verifyIsEmptyBlock = function (blockContentValues) {
```

Comment:
Maybe rename the arg to defaultBlockContentValues or something like that. Actually I wonder if the default should be part of the testCases structure. Something like:

```json
{
    defaultBlockContentValues: [],
    testCases: []
}
```

As it stands now, the test cases are assuming a specific default value but these are actually passed in to the tester function when it is called.